pyramid/scaffolds/alchemy/development.ini_tmpl

```diff
@@ -26,7 +26,7 @@ sqlalchemy.url = sqlite:///%(here)s/{{project}}.sqlite
 
 [server:main]
 use = egg:waitress#main
-listen = 127.0.0.1:6543 [::1]:6543
+listen = localhost:6543
```

Comment:
We have deprecated the scaffolds, so you can revert changes to this scaffold and the other two.